### PR TITLE
task: Add new filters

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: letter_templates/views/view.py [hunk 1]
```diff
@@ -6,13 +6,23 @@
...
+        name = request.GET.get("name")
+        if name:
+            params["name"] = name
+
         data, _ = get_letter_templates(request, convert_dict_to_query_params(params))
-        context = {"data": data, "page": params.pop("page")}
+        filters = FiltersBar([TextInput(name="name", title="name"),])
+        context = {
+            "data": data,
+            "page": params.pop("page"),
```
Comment (jfaracik) [inline letter_templates/views/view.py:23]: Do we still need page in context?

Comment (jfaracik) [reply letter_templates/views/view.py:23]: + could params not be `params = {"page": int(request.GET.get("page", 1), "name": ...)}`

File: users/views/users.py [hunk 2]
```diff
@@ -23,9 +23,12 @@
 class UsersList(TemplateView):
     def get(self, request, **kwargs):
         status = self.request.GET.get("status")
+        email = self.request.GET.get("email")
         params = {"page": int(self.request.GET.get("page", 1))}
```
Comment (jfaracik) [inline users/views/users.py:27]: Same comment as above.

Comment (BenAndrew-Kainos) [reply users/views/users.py]: done

File: letter_templates/views/view.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply letter_templates/views/view.py]: done

File: letter_templates/views/view.py [hunk 3]
```diff
@@ -6,13 +6,19 @@
...
 )
+from lite_forms.components import FiltersBar, TextInput
 
 
 class LetterTemplatesList(TemplateView):
     def get(self, request, **kwargs):
-        params = {"page": int(request.GET.get("page", 1))}
+        params = {"page": int(request.GET.get("page", 1)), "name": request.GET.get("name", "")}
+
         data, _ = get_letter_templates(request, convert_dict_to_query_params(params))
```
Comment (jfaracik) [inline letter_templates/views/view.py:16]: Use convert_parameters_to_query_params instead of convert_dict_to_query_params :)

Comment (jfaracik) [reply letter_templates/views/view.py:16]: Brownie points for doing `convert_parameters_to_query_params` inside of `get_letter_templates` instead

Comment (BenAndrew-Kainos) [reply letter_templates/views/view.py]: done

**Pull request merged**